apps/web/app/root.tsx

```diff
@@ -7,17 +7,13 @@ import leatherUiStyles from '@leather.io/ui/styles?url';
 import type { Route } from './+types/root';
 import stylesheet from './app.css?url';
 
-/**
- * Polyfill global Buffer
- * (should be called before another code)
- */
+// Polyfill global Buffer
+// (should be called before another code)
```

Comment:
I expect better grammar from you here

Reply:
This wasn't me, but yes I should have fixed haha